cpp/sort/bubble_sort.cpp

```diff
@@ -0,0 +1,40 @@
+/* 
+Implementation of Bubble Sort in C++ (gcc 4.9.2)
+https://en.wikipedia.org/wiki/Bubble_sort
+*/
+
+#include <iostream>
+using namespace std;
+
+int array[100000];
+
+void bubble_sort(int n)
+{
+	int temp,i,j;
+	for(i=0;i<n;i++)
+	{
+		for(j=0;j<(n-i-1);j++)
+		{
+			if(array[j]>array[j+1])
+			{
+				temp=array[j];
+				array[j]=array[j+1];
+				array[j+1]=temp;
+			}
+		}
+	}
+}
+
+int main()
```

Comment:
Keep tests in separate function like tester/checker etc. Call that function in main for testing.